src/zenml/integrations/modal/__init__.py

```diff
@@ -11,25 +11,26 @@
 #  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
 #  or implied. See the License for the specific language governing
 #  permissions and limitations under the License.
-"""Modal integration for cloud-native step execution.
+"""Modal integration for cloud-native step execution and orchestration.
 
-The Modal integration sub-module provides a step operator flavor that allows
-executing steps on Modal's cloud infrastructure.
+The Modal integration sub-module provides a step operator flavor and an orchestrator
+flavor that allow executing steps and complete pipelines on Modal's cloud infrastructure.
 """
 from typing import List, Type
 
 from zenml.integrations.constants import MODAL
 from zenml.integrations.integration import Integration
 from zenml.stack import Flavor
 
+MODAL_ORCHESTRATOR_FLAVOR = "modal"
 MODAL_STEP_OPERATOR_FLAVOR = "modal"
 
 
 class ModalIntegration(Integration):
     """Definition of Modal integration for ZenML."""
 
     NAME = MODAL
-    REQUIREMENTS = ["modal>=0.64.49,<1"]
+    REQUIREMENTS = ["modal>=1"]
```

Comment:
Have you tested that this doesn't break the step operator?

Reply:
have not checked, thats one thing id love your help on @strickvl

Reply:
Checked it and it works